tests/e2e/utils/README.md

```diff
@@ -153,6 +153,34 @@ This package provides support for enabling retries in tests:
 | `updatePaymentGateway`| `paymentGatewayId`, `payload` | Update a payment gateway |
 | `getSystemEnvironment` | | Get the current environment from the WooCommerce system status API.
 
+### Classes
+
+The package includes the following page specific utility classes
```

Comment:
Just a minor suggestion to add a colon at the end:

```suggestion
The package includes the following page specific utility classes:
```

Reply:
Fixed in https://github.com/woocommerce/woocommerce/pull/30696/commits/5ccf3a564fb4cc4cb4c392584320c6e6ec20069d